nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -547,14 +548,25 @@ function AddSubscriberDetails(props: DialogProps) {
             state: subscriber.state,
             sub_profile: subscriber.dataPlan,
           },
-        });
+        };
         successCountRef.current = successCountRef.current + 1;
+        addedSubscribers.push(newSubscriber);
       } catch (e) {
         const errMsg = e.response?.data?.message ?? e.message ?? e;
         setError('error saving ' + subscriber.imsi + ' : ' + errMsg);
         return;
       }
     }
+    //bulk add subscribers at the end
+    if (addedSubscribers.length > 0) {
+      try {
+        await ctx.setState?.('', addedSubscribers);
+      } catch (e) {
+        const errMsg = e.response?.data?.message ?? e.message ?? e;
+        setError('error saving subscribers to the api : ' + errMsg);
+        return;
+      }
+    }
     enqueueSnackbar(
```

Comment:
You can get rid of LinearProgressWithLabel since we don't know about the progress here.

Reply:
Instead of completely removing the error message, changed it to show the error on the Snackbar incase the communication with the API fails, so that the user knows the operation failed.

Reply:
Completely removed Linear Progress and replaced by snackbar as per our chat